pkg/kubectl-argo-rollouts/info/rollout_info.go

```diff
@@ -326,3 +329,35 @@ func (r *RolloutInfo) AnalysisRunsByRevision(rev int) []AnalysisRunInfo {
 	}
 	return runs
 }
+
+func addSteps(stepsIndex int, index *int32, stepsData *[]string, steps v1alpha1.CanaryStep, ro *v1alpha1.Rollout) {
+	currentIcon := ""
+	if int32(stepsIndex)+1 == *index && *index <= int32(len(ro.Spec.Strategy.Canary.Steps)) {
+		currentIcon = "[*]"
+	}
+	if steps.SetWeight != nil {
+		*stepsData = append(*stepsData, fmt.Sprintf("%vsetWeight:%v", currentIcon, *steps.SetWeight))
+	}
+	if steps.Pause != nil {
+		pause := fmt.Sprintf("%vpause:always", currentIcon)
```

Comment:
Can we use `∞` (infinity) to indicate an indefinite pause?

Reply:
Pretty good!

Reply:
I done!